dolphinscheduler-dao/src/main/resources/datasource.properties

```diff
@@ -16,16 +16,16 @@
 #
 
 # datasource configuration
-spring.datasource.driver-class-name=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://127.0.0.1:5432/dolphinscheduler
-spring.datasource.username=root
-spring.datasource.password=root
+#spring.datasource.driver-class-name=org.postgresql.Driver
+#spring.datasource.url=jdbc:postgresql://127.0.0.1:5432/dolphinscheduler
+#spring.datasource.username=root
+#spring.datasource.password=root
 
 # mysql example
-#spring.datasource.driver-class-name=com.mysql.jdbc.Driver
-#spring.datasource.url=jdbc:mysql://127.0.0.1:3306/dolphinscheduler?useUnicode=true&characterEncoding=UTF-8
-#spring.datasource.username=ds_user
-#spring.datasource.password=dolphinscheduler
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url=jdbc:mysql://127.0.0.1:3306/dolphinscheduler?useUnicode=true&characterEncoding=UTF-8
+spring.datasource.username=ds_user
+spring.datasource.password=dolphinscheduler
```

Comment:
@CalvinKirs we'd better provide Spring profiles to allow developers to specify different profile when developing, instead of modifying these over and over, although I think this should not be submitted in this PR